### DISCUSSION TITLE
How to create a subnet within the project factory?

### DISCUSSION THREAD

**Discussion by 1tsygo** [GoogleCloudPlatform/cloud-foundation-fabric#2161] [Q&A] [answered]
Hi,

I've recently created a new project using the project factory. I'm now curious if it's possible to create a subnet during the project creation process. If it is, could you please guide me on how to do so? I've searched but couldn't find any resources to help me with this.

Could you also provide an example? :)

Regards,
1tsygo

**Comment by ludoo** [comment, marked as answer]
It's not possible, subnets have their own factory in the net-vpc module. It's really easy to combine them if you need to, just add the module to the project factory and one more attribute to the factory configs variable.
